packages/site/src/components/Navigation/HomeLink.tsx

```diff
@@ -6,19 +6,21 @@ export function HomeLink({
   logoDark,
   logoText,
   name,
+  url,
 }: {
   logo?: string;
   logoDark?: string;
   logoText?: string;
   name?: string;
+  url?: string;
 }) {
   const Link = useLinkProvider();
   const baseurl = useBaseurl();
   const nothingSet = !logo && !logoText;
   return (
     <Link
       className="flex items-center ml-3 dark:text-white w-fit md:ml-5 xl:ml-7"
-      to={withBaseurl('/', baseurl)}
+      to={url ? url : withBaseurl('/', baseurl)}
```

Comment:
@stefanv I'm on mobile, so not super able to dig in to the code. Is this how we also handle the nav links? It seems like we might want to apply base url handling to the logo url too.

I know at one point we might not have included the base url with nav links, which needs (needed?) to be fixed!